server/app/views/applicant/ApplicantBaseFragment.html

```diff
@@ -0,0 +1,24 @@
+<head th:fragment="pageHeader">
+    <meta charset="UTF-8">
+    <title>Title</title>
+
+    <link th:href="${tailwindStylesheet}" type="text/css" rel="stylesheet">
+    <link th:href="${uswdsStylesheet}" type="text/css" rel="stylesheet">
+    <script th:src="${uswdsJsInit}" type="text/javascript"></script>
+    <script th:src="${adminJsBundle}" type="text/javascript"></script>
+</head>
+
+<footer th:fragment="pageFooter">
+    <script th:src="${uswdsJsBundle}" type="text/javascript"></script>
+</footer>
+
+<div th:fragment="requiredFieldsExplanation">
+  <!-- This uses utext because the parameter passed into the message is html,
+    it is a placeholder span with the red asterisk. We need it to be html and not
+    raw text to recreate the red style -->
+  <p th:utext="#{content.requiredFieldsNote(~{applicant/ApplicantBaseFragment :: requiredAsterisk})}"></p>
+</div>
+
+<th:block th:fragment="requiredAsterisk">
+    <span class="text-red-600 font-semibold"> *</span>
```

Comment:
It seems like this would be easier as a shared css class we can just use on a `<span class="required-asterisk"> *</span>` in each question, instead of a template fragment we pull into every question

Reply:
Yes that would certainly be better. I actually didn't know we used any css classes directly, server/app/assets/stylesheets/styles.css seems like the only place, is that right?

Reply:
I _think_ that's the only place, outside of the USWDS classes. tailwind also has a way to wrap a few tailwind classes in an outer class.

looks like there might actually be a uswds class that applies? `usa-hint usa-hint--required` based on the form example:
https://designsystem.digital.gov/templates/form-templates/address-form/

Reply:
So we can do that, but we still need the template to use as a value passed into the requiredFieldsExplanation. Might be worth just using the span in other usages though